zipline/data/data_portal.py

```diff
@@ -0,0 +1,1186 @@
+#
+# Copyright 2015 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+import bcolz
+from logbook import Logger
+
+import numpy as np
+import pandas as pd
+
+from zipline.assets import Asset
+
+from zipline.utils import tradingcalendar
+from zipline.errors import (
+    NoTradeDataAvailableTooEarly,
+    NoTradeDataAvailableTooLate
+)
+
+# FIXME anything to do with 2002-01-02 probably belongs in qexec, right/
+FIRST_TRADING_DAY = pd.Timestamp("2002-01-02 00:00:00", tz='UTC')
```

Comment:
Found the hardcoded value you mentioned.  Just commenting so we can easily find it again!